include/configs/s5p_aries.h

```diff
@@ -80,6 +80,13 @@
 	"stdin=serial,gpio-keys\0" \
 	"stdout=serial,vidconsole\0" \
 	"stderr=serial,vidconsole\0" \
+	"mmcdev=0\0" \
+	"mmcpart=1\0" \
+	"loadkernel="\
+		"ext4load mmc ${mmcdev}:${mmcpart} ${kernel_load_addr} uImage "\
```

Comment:
Since the bootm ${kernel_load_addr} is common to all the MMC boot, should it be added to loadkernel as well (with loadkernel renamed to something like mmc_boot)?

Reply:
Ok, i'll change it. Just give me a few minutes.